migrations/2017_09_17_100000_create_role_user_table.php

```diff
@@ -23,7 +23,7 @@ public function up()
                 $userForeignKey = $userModel->getForeignKey();
 
                 $table->unsignedInteger('role_id');
-                $table->unsignedInteger($userForeignKey);
+                $table->unsignedBigInteger($userForeignKey);
```

Comment:
We could consider moving all ids and foreign ids to use `unsignedBigInteger`, or use the more modern schema helpers